website/docs/secrets/getting-started.mdx

```diff
@@ -2,3 +2,188 @@
+
+<AlphaWarning/>
+
+This guide shows you a basic experience to get started with Weave Gitops Secrets.
+It covers the scenario of setting up the capability in a test environment and how to use it for your applications.
+
+## Requirements
+- You have a test Weave Gitops Enterprise environment with Flux installed.
+- You have a secret in AWS secrets manager.
+
+## Add the secrets infra
+
+In order to allow you managing external secrets stores and secrets, install `external-secrets` profile
+from `weaveworks-charts` profiles repository.
+
+<details><summary>Expand to see example</summary>
+
+```yaml
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: weaveworks-charts
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  provider: generic
+  timeout: 1m0s
+  url: https://weaveworks.github.io/weave-gitops-profile-examples/
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  labels:
+    weave.works/applied-layer: layer-0
+  name: external-secrets
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  chart:
+    spec:
+      chart: external-secrets
+      sourceRef:
+        apiVersion: source.toolkit.fluxcd.io/v1beta2
+        kind: HelmRepository
+        name: weaveworks-charts
+        namespace: flux-system
+  install:
+    crds: CreateReplace
+    createNamespace: true
+```
+</details>
+
+And see it available in your cluster
+
+![setup infra](imgs/getting-started-setup-infra.png)
+
+## Connect your secrets store
+
+In order to connect with your secrets stores, install `secrets-store-config` profile
+from `weaveworks-charts` profiles repository with the values for your store.
+
+<details><summary>Expand to see example</summary>
+
+```yaml
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  labels:
+    weave.works/applied-layer: layer-0
+  name: secrets-store-config
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  chart:
+    spec:
+      chart: secrets-store-config
+      sourceRef:
+        apiVersion: source.toolkit.fluxcd.io/v1beta2
+        kind: HelmRepository
+        name: weaveworks-charts
+        namespace: flux-system
+  values:
+    secretStores:
+     aws-secrets-manager:
+         provider:
+           aws:
+             service: SecretsManager
+             region: eu-north-1
+             auth:
+               secretRef:
+                 accessKeyIDSecretRef:
+                   name: awssm-secret
+                   key: access-key
+                 secretAccessKeySecretRef:
+                   name: awssm-secret
+                   key: secret-access-key
+```
+</details>
+
+And see it available in your cluster
+
+![setup secret stores](imgs/getting-started-setup-config.png)
+
+## Create the secret
+
+Given you have a secret in AWS Secrets Manager for example `test/search/db`.
+
+![aws secret](imgs/getting-started-secret-asm.png)
+
+Create the External Secret manifest to pull the secret from your store into your environment.
```

Comment:
cant we point the user here to the page covering the UI? our UI already does this for the user. Or instead of pointing the user to the other page, have a minimal version of it in this section instead. Unless if you think there's a certain value in showing the user how to do it using the manifests.

Reply:
good call, updated! let me know whether it makes sense